orchagent/portsorch.cpp

```diff
@@ -2013,7 +2061,38 @@ void PortsOrch::doVlanMemberTask(Consumer &consumer)
             }
 
             if (addBridgePort(port) && addVlanMember(vlan, port, tagging_mode))
```

Comment:
could you add this portion of the code into `addBridgePort` function and set the attribute during the bridge port creation time?

Reply:
Good point. we will refine code per your suggestion.